configTokens.json

```diff
@@ -0,0 +1,63 @@
+{
+    "supportAdministrators":false,
+    "supportVerification":false,
+    "integration":"WickrIO-Web-Interface",
+    "tokens":[
+        {
+            "token": "port",
```

Comment:
I did not mean change the Token value, I meant change the "type" value to "port"